codegen.ts

```diff
@@ -1,7 +1,7 @@
 import { CodegenConfig } from '@graphql-codegen/cli';
 
 const config: CodegenConfig = {
-  schema: process.env.REACT_APP_API_URI,
+  schema: process.env.VITE_API_URI,
```

Comment:
This is in `master` branch already so just rebasing will get rid of this. But it won't probably hurt either if you merge this as is as then it'll just throw your change away as already applied I think.

Reply:
ok, i thought i had up to date master